js/src/lib/util.js

```diff
@@ -144,16 +144,32 @@ function parseHighlightedAttribute({
   return elements;
 }
 
-export function packageJSONLink({ githubRepo }) {
-  if (githubRepo) {
-    const { user, project, path, head } = githubRepo;
+export function packageJSONLink({ repository }) {
+  if (!repository) {
```

Comment:
WDYT if we linked to unpkg/jsdelivr? less logic and the direct file, + it will work without repository field

Reply:
I think it would be better, I'll do this.